test/setup/platforms/kind-olm.ts

```diff
@@ -0,0 +1,27 @@
+import { createCluster as kindCreateCluster } from './kind';
+import * as kubectl from '../../helpers/kubectl';
+import * as sleep from 'sleep-promise';
+
+export async function createCluster(version: string): Promise<void> {
+  await kindCreateCluster(version);
+
+  // OLM installation: https://github.com/operator-framework/community-operators/blob/master/docs/testing-operators.md#2-install-olm
+
+  const operatorLifecycleManagerVersion = '0.15.1'; // https://github.com/operator-framework/operator-lifecycle-manager/releases/tag/0.15.1
+  await kubectl.applyK8sYaml(`https://github.com/operator-framework/operator-lifecycle-manager/releases/download/${operatorLifecycleManagerVersion}/crds.yaml`);
+  await sleep(5000); // give enough time to k8s to apply the previous yaml
+  await kubectl.applyK8sYaml(`https://github.com/operator-framework/operator-lifecycle-manager/releases/download/${operatorLifecycleManagerVersion}/olm.yaml`);
+  await kubectl.waitForDeployment('catalog-operator', 'olm');
+  await kubectl.waitForDeployment('olm-operator', 'olm');
+  // await kubectl.waitForDeployment('operatorhubio-catalog', 'olm');
```

Comment:
Do we need this catalog, or it is optional?

Reply:
We don't need it. I just forgot to delete it!